lib/tasks/ci.rake

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 desc 'Runs the continuous integration scripts'
-task ci: %i[lint security spec:with_codeclimate_coverage]
+task ci: %i[lint spec:with_codeclimate_coverage]
```

Comment:
We can't remove security checks. Please revert this.

Reply:
The reason we took this out is because we need to get this out and tested, I am running into an issue where its telling me that Ruby 2.6.6 will be depreciated as of March 31 2022, I attempted bumping it up to Ruby 2.7.0 and Ruby 2.7.4 and was still getting the same error pointing to the rails gems. I tried to update those and it seemed liked a large undertaking.

Reply:
It should be updated with ruby 2.7.4 and I added back in the security check